File: Marriage/MarriageApi/Marriage.Entity/Data/ServiceInterface.cs
using OpenDataAccessCore.Entity;
using System;
using System.Collections.Generic;
using System.Text;

namespace Marriage.Entity.Data
{
    /// <summary>
    /// 服务接口
    /// </summary>
    [TableProperty(Name = "t_ServiceInterface", PrimaryKey = "ServiceInterfaceId")]
    public class ServiceInterface : EntityModel, IEntity
    {
        /// <summary> 
        /// 主键
        /// </summary> 
        public Guid ServiceInterfaceId { get; set; }
        /// <summary> 
        /// 名称
        /// </summary> 
        public string Name { get; set; }
        /// <summary> 
        /// 接口名
        /// </summary> 
        public string InterfaceName { get; set; }
        /// <summary> 
        /// 方法名
        /// </summary> 
        public string MethodName { get; set; }
        /// <summary> 
        /// URL
        /// </summary> 
        public string Url { get; set; }
        /// <summary> 
        /// 备注
        /// </summary> 
        public string Remark { get; set; }
        /// <summary> 
        /// 是否删除
        /// </summary> 
        public byte IsDelete { get; set; }
        /// <summary> 
        /// 创建人
        /// </summary> 
        public Guid CreateUser { get; set; }
        /// <summary> 
        /// 创建时间
        /// </summary> 
        public DateTime CreateDate { get; set; }
        /// <summary> 
        /// 更新人
        /// </summary> 
        public Guid UpdateUser { get; set; }
        /// <summary> 
        /// 更新时间
        /// </summary> 
        public DateTime UpdateDate { get; set; }
        /// <summary> 
        /// 行版本
        /// </summary> 
        public string RowVersion { get; set; }
    }
}

File: Marriage/MarriageWeb/page-configs/entities/serviceInterface.js
module.exports = {
    name: 'ServiceInterface',
    viewName: 'ViewServiceInterface',
    primaryKey: 'ServiceInterfaceId',
    properties: getProperties()
  }
  
  function getProperties() {
    return [
        getProperty('ServiceInterfaceId', 'ServiceInterfaceId'),
        getProperty('Name', '名称'),
        getProperty('InterfaceName', '接口名'),
        getProperty('MethodName', '方法名'),
        getProperty('Url', 'URL'),
        getProperty('Remark', '备注'),
        getProperty('CreateDate', '创建时间')
    ]
  }
  
  function getProperty(name, label) { return { name, label } }